Build failure in 1.0.4: undefined reference to `opj_t1_ht_decode_cblk'

Building openjpeg-sys 1.0.4 [fails in my project](https://github.com/noritada/grib-rs/issues/16) with a following error:

```
/path/to/openjpeg-sys-1.0.4/vendor/src/lib/openjp2/t1.c:1690: undefined reference to `opj_t1_ht_decode_cblk'
```

I will send a PR later.

--- a/build.rs
+++ b/build.rs
@@ -21,6 +21,7 @@ fn main() {
         "cio.c",
         "dwt.c",
         "event.c",
+        "ht_dec.c",
         "image.c",
         "invert.c",
         "j2k.c",

